### ISSUE TITLE
@SjoenH Can you add last updated/modified update when To-be process has changes as a requirement? Ref #258

### ISSUE THREAD

**Issue by SjoenH** [equinor/MAD-VSM-WEB#282]
@SjoenH Can you add last updated/modified update when To-be process has changes as a requirement? Ref #258

_Originally posted by @knudsvik in https://github.com/equinor/MAD-VSM-WEB/issues/192#issuecomment-961714547_

**Comment by Magos** [comment]
@SjoenH @martabenthem Would it get in the way of the front-end if these object fields
![firefox_155znCnlsn](https://user-images.githubusercontent.com/1004386/140501086-9f98d9b9-d15d-4f1b-b4b3-010a33768a01.png)
got replaced with just the datetimes? I'm thinking of adding these queries as virtual columns in SQL.

**Comment by SjoenH** [comment]
Not really. We don't use it today, but #245 could need something like it.

**Comment by martabenthem** [comment]
I don't use any of those objects directly. As long as the sorting by "date created" and "last modified" continues to work in the search query, seems good to me.
